src/bin/ion/commands/jq.rs

```diff
@@ -173,11 +206,141 @@ impl PartialOrd for JaqElement {
 
 // === Math operator behaviors ===
 
+trait DecimalMath {
+    fn to_big_decimal(self) -> BigDecimal;
+    fn to_decimal(self) -> Decimal;
+    fn add(self, v2: impl DecimalMath) -> Decimal
+    where
+        Self: Sized,
+    {
+        let d1 = self.to_big_decimal();
+        let d2 = v2.to_big_decimal();
+        (d1 + d2).to_decimal()
+    }
+}
```

Comment:
Any organization suggestions for these DecimalMath/FloatMath traits would be appreciated!